package.json

```diff
@@ -20,17 +20,45 @@
     "url": "https://github.com/techswitch-learners/testswitch-frontend-candidate/issues"
   },
   "homepage": "https://github.com/techswitch-learners/testswitch-frontend-candidate#readme",
+  "jest": {
+    "preset": "ts-jest",
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json",
+      "node"
+    ],
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "setupFilesAfterEnv": [
+      "<rootDir>/_test_/setupTests.ts"
+    ],
+    "moduleNameMapper": {
+      "^.+\\.(css|less|scss)$": "babel-jest"
+    }
+  },
   "dependencies": {
     "@zeit/next-sass": "^1.0.1",
     "next": "^9.3.4",
     "react": "^16.13.1",
     "react-dom": "^16.13.1"
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.5.0",
+    "@testing-library/react": "^10.0.2",
+    "@types/jest": "^25.2.1",
     "@types/node": "^13.11.1",
     "@types/node-sass": "^4.11.0",
     "@types/react": "^16.9.34",
+    "@types/testing-library__react": "^10.0.1",
+    "@zeit/next-sass": "^1.0.1",
```

Comment:
this is now in `dependencies`. shouldn't also need it in `devDependencies`

Reply:
I've made the change, should be good now